signature/policy_eval_sigstore_test.go

```diff
@@ -89,8 +89,13 @@ func TestPRSigstoreSignedFulcioPrepareTrustRoot(t *testing.T) {
 
 func TestPRSigstoreSignedPrepareTrustRoot(t *testing.T) {
 	const testKeyPath = "fixtures/cosign.pub"
+	const testKeyPath2 = "fixtures/cosign.pub"
```

Comment:
I think we want two truly different paths with different contents to test correctness of the code.